rome.json

```diff
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://docs.rome.tools/schemas/12.1.3/schema.json",
+	"organizeImports": {
+		"enabled": false
+	},
+	"files": {
+		"ignore": ["built/", "**/browser_polyfill.js"]
+	},
+	"linter": {
+		"enabled": true,
+		"rules": {
+			"recommended": true,
+			"complexity": {
+				"noExtraBooleanCast": "off"
```

Comment:
I personally think `!Boolean(variable)` is easier to read/understand than `!!variable`, but if you don't agree, we just disable this rule and run `npm run format`.